src/docs/docusaurus.config.js

```diff
@@ -10,8 +10,8 @@ const config = {
   tagline: "A tool for tracking blogs in orbit around Seneca's open source involvement",
   url: 'https://telescope.cdot.systems',
   baseUrl: '/',
-  onBrokenLinks: 'throw',
-  onBrokenMarkdownLinks: 'warn',
+  onBrokenLinks: 'ignore',
+  onBrokenMarkdownLinks: 'ignore',
```

Comment:
Should we ignore that? I think we can create a follow-up issue to fix the broken links

Reply:
If you file that issue, sure.

Reply:
https://github.com/Seneca-CDOT/telescope/issues/3048 Follow up issue